src/main/java/frc/robot/subsystems/ShooterAdjuster.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkMax;
+import com.spikes2212.command.genericsubsystem.smartmotorcontrollersubsystem.SparkGenericSubsystem;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import frc.robot.RobotMap;
+
+public class ShooterAdjuster extends SparkGenericSubsystem {
```

Comment:
Add the ratio of motor rotations to one degree angle change. It's not exactly a gear ratio, so figure out a name. Also configure the relative encoder to abide by that ratio.